nest-demo/otel-demo/docker-run.sh

```
@@ -0,0 +1,4 @@
+docker stop otel-demo
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)